modules/appeals_api/config/schemas/supplemental_claims/v0/evidence_submission.json

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
I added this schema to help validate the body that the evidence submission `create` endpoint now expects. I use it in with the `form_schemas` instance in the evidence submission controller (see my note there)